.github/workflows/tests.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     container: python:${{ matrix.python }}
     strategy:
       matrix:
-        python: [3.8, 3.9, 3.10.0]
+        python: [3.8, 3.9, 3.10]
```

Comment:
it`s not just how it was done;) Zero is discarded with this type of record "3.10"

Reply:
Ohhh, I will add it back.

Reply:
Done